src/main/java/org/openmrs/isanteplus/performancedata/generator/service/GeneratorService.java

```diff
@@ -0,0 +1,33 @@
+package org.openmrs.isanteplus.performancedata.generator.service;
+
+import org.openmrs.isanteplus.performancedata.generator.service.impl.PatientGeneratorService;
+import org.openmrs.isanteplus.performancedata.model.AbstractEntity;
+import org.openmrs.isanteplus.performancedata.model.connection.Inserter;
+import org.openmrs.isanteplus.performancedata.options.GeneratorOptions;
+import org.springframework.stereotype.Service;
+
+import javax.inject.Inject;
+import java.sql.SQLException;
+import java.util.Set;
+
+@Service
+public class GeneratorService {
+
+    @Inject
+    private PatientGeneratorService patientService;
+
+    public void genereteDatabase(GeneratorOptions options) throws SQLException {
+
+        Inserter ins = new Inserter(options);
+        ins.connect();
```

Comment:
this connection throws the same exception as well, so maybe put it in to try statement?

Reply:
ok